frontend/src/components/VariantListPage/VariantsTable.tsx

```diff
@@ -359,21 +368,28 @@ const BASE_COLUMNS: ColumnDef[] = [
     key: "an",
     heading: "Allele number",
     isNumeric: true,
+    width: 120,
     sortKey: (variant) => variantAN(variant),
-    render: (variant) => renderCount(variantAN(variant)),
+    render: (variant) => (
+      <Flex ml="auto">{renderCount(variantAN(variant))}</Flex>
+    ),
   },
   {
     key: "af",
     heading: "Allele frequency",
     isNumeric: true,
+    width: 140,
     sortKey: (variant) => variantAF(variant),
-    render: (variant) => renderAlleleFrequency(variantAF(variant)),
+    render: (variant) => (
+      <Flex ml="auto">{renderCount(variantAF(variant))}</Flex>
```

Comment:
We should use the helper that was used here previously, '`renderAlleleFrequency`' to render this is scientific notation, this other helper '`renderCount`' rounds to the nearest whole number, so all the AFs are rendering as '`0`'